core/src/avm2/globals/string.rs

```diff
@@ -92,6 +92,18 @@ pub fn class_init<'gc>(
     Ok(Value::Undefined)
 }
 
+pub fn call_handler<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    _this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    Ok(args
+        .get(0)
+        .unwrap_or(&Value::Undefined)
```

Comment:
```
var cls = String;
trace(cls()); // ""
```
So this should be a `.unwrap_or(&Value::String("".into()))`, I think

Reply:
Indeed, thanks!